LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 ChainSquad GmbH
+Copyright (c) 2018 Codaone Oy
```

Comment:
ChainSquad GmbH copyright can't just be removed because of MIT license. 
This could be changed to something like:
```Copyright for portions of project DEXBot are held by ChainSquad GmbH 2017 as part of project stakemachine. All other copyright for project DEXBot are held by Codaone Oy 2018.```